### DISCUSSION TITLE
Where is entire api document?

### DISCUSSION THREAD

**Discussion by Winlchance** [MonoGame/MonoGame#8133] [General]
api document in https://monogame.net/api/ seems incomplete.many class such as Game, Texture can't be found.

**Comment by Apostolique** [comment]
That issue is tracked here: https://github.com/MonoGame/monogame.github.io/issues/58.

**Comment by AristurtleDev** [comment]
Pull request https://github.com/MonoGame/monogame.github.io/pull/91 was put in that should resolve this issue.

**Reply by AristurtleDev** [reply]
@WinlCheng this should now be resolved with the most recent PR pushed through.

**Reply by Winlchance** [reply]
yes, it seems work correctly
